metric-publishers/emf-metric-publisher/src/test/java/software/amazon/awssdk/metrics/publishers/emf/EmfMetricPublisherTest.java

```diff
@@ -38,14 +39,15 @@ void setUp() {
 
     @Test
     void Publish_NoLogGroupName() {
-        try {
+        assertThatThrownBy(() -> {
             EmfMetricPublisher publisher = publisherBuilder.build();
             publisher.publish(null);
-        } catch (Exception e) {
-            assertThat(e).hasMessage("logGroupName must be configured for publishing emf format log");
-        }
+        })
+            .isInstanceOf(Exception.class)
```

Comment:
Exception.class is a pretty generic type and we should probably assert the exact type in tests, suggesting `NullPointerException`